### ISSUE TITLE
Add a new page for showcasing projects, projects.html

### ISSUE THREAD

**Issue by mahakmakharia** [dsc-iem/Website#28]
Create a new page where we can list the projects made by our members. 

It will include a list of projects where each project will have details like
1. Name of the project
2. Name of the creator(s)
3. Relevant image/video of the project
4. Link to the source code/website/app (if live)

It will also include a section where students can request their project to be showcased. Add details like
1. Small text
2. Button which will redirect to a new form

You can either do it from the beginning to the end or a part of it. Let me know!

**Comment by the-agarwal01** [comment]
Hey,
Can you assign us this issue? It looks good for us.I along with @damikdhar are planning to do the same!

**Comment by damikdhar** [comment]
Yes it would be great for me to work with @the-agarwal01 in this project!

**Comment by mahakmakharia** [comment]
Okay then both of you can collectively work on this and work on separate parts so that you both can contrbute
